data/mods/Magiclysm/items/mana_crystals.json

```
@@ -13,7 +13,7 @@
     "name": { "str_sp": "crystallized mana" },
     "symbol": "=",
     "color": "yellow",
-    "description": "Some crystallized mana.  This can be reloaded into rechargable mana crystals, but can never be unloaded.",
+    "description": "Some crystallized mana.  This can be reloaded into rechargeable mana crystals, but can never be unloaded.",
```

@KorGenT: "reloaded" or "loaded"?

---

... yes? i'm not sure the difference matters here

---

Reloaded implies they were unloaded from something.
```suggestion
    "description": "Some crystallized mana.  This can be loaded into rechargeable mana crystals, but can never be unloaded.",
```